src/datajudge/constraints/stats.py

```diff
@@ -0,0 +1,47 @@
+from typing import Any, Optional, Tuple
+
+import sqlalchemy as sa
+from scipy.stats import ks_2samp
+
+from .. import db_access
+from ..db_access import DataReference
+from .base import Constraint, OptionalSelections
+
+
+class KolmogorovSmirnov2Sample(Constraint):
+    def __init__(
+        self, ref: DataReference, ref2: DataReference, significance_level: float = 0.05
+    ):
+        self.significance_level = significance_level
+        super().__init__(ref, ref2=ref2)
+
+    @staticmethod
+    def calculate_2sample_ks_test(data, data2):
+        """
+        For two given lists of values calculates the Kolmogorov-Smirnov test.
+        Read more here: https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.kstest.html
+        """
+
+        # calculate statistic
```

Comment:
I'd say that this comment is rather superfluous as it is now. Maybe you could add on why taking this approach and some of its potential failings. And even going further, mentioning at least the first level improvement of letting the engine do the CDF.